Source/core/Process.h

```diff
@@ -560,7 +560,9 @@ namespace Core {
                 TerminateProcess(_info.hProcess, 1234);
             }
 #else
-            ::kill(_PID, (hardKill ? SIGKILL : SIGTERM));
+            ASSERT((errno = 0) == 0);
```

Comment:
Why is this in here? Is there a very specific reason? Resetting should not be needed as far as I know (and if so now it is only done in debug) and why then check if it worked?
Just remove this line?